sass/base/core/_base-reset.scss

```diff
@@ -31,7 +31,6 @@ body {
 	-webkit-font-smoothing: antialiased;
 }
 
-picture img,
-legend {
```

Comment:
I think making the legend 100% wide was just put in by Andy to resolve an IE issue